lib/src/rave_pay_manager.dart

```diff
@@ -36,10 +36,10 @@ class RavePayManager {
   ///
   /// Please, enable embedded_views_preview on iOS. See https://stackoverflow.com/a/55290868/6181476
   ///  {@endtemplate}
-  Future<RaveResult> prompt({
-    @required BuildContext context,
-    @required RavePayInitializer initializer,
-  }) async {
+  Future<RaveResult> prompt(
+      {@required BuildContext context,
+      @required RavePayInitializer initializer,
+      @required Function(RaveResult raveResult) onTransactionComplete}) async {
```

Comment:
We don't need this. please. `prompt()` returns a future that completes with `RaveResult`. `RaveResult` contains the transaction result after a successful or failed payment.

Reply:
The reason i used a callback is because the prompt wasn't returning my **RaveResult** it was logging null ... but it i will revert to the promt & send another P.R

Reply:
Really? Does this happen for an error or success transaction or both?